### DISCUSSION TITLE
does corkey delete  data?

### DISCUSSION THREAD

**Discussion by Ender-Ghost-Craft** [3kh0/ext-remover#826] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
does it delete downloads?

**Comment by AshtonDavies** [comment, marked as answer]
Yes, Corkey does indeed include power washing the Chromebook, which wipes local data including everything under "My files," so I suggest you select everything you want to drag and back up to Google Drive if that's available for your account.

**Reply by Ender-Ghost-Craft** [reply]
ok thank you, maybe add  that to the readme?
